common/changes/@uifabric/charting/code-formatting_2018-09-24-22-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/charting",
+      "comment": "Moving tslint/prettier dependencies ",
+      "type": "patch"
+    }
```

Comment:
All changes to all packages are dev-dependencies only and have 0 impact on consuming apps. Consequently these should technically all be of type none. I recommend doing a find and replace.